spec/models/miq_provision_spec.rb

```diff
@@ -4,7 +4,7 @@
   context "A new provision request," do
     before(:each) do
       @os = OperatingSystem.new(:product_name => 'Microsoft Windows')
-      @admin = FactoryGirl.create(:user_admin)
+      @admin = FactoryGirl.create(:user_with_group, :role => "admin")
```

Comment:
yea - maybe the old code should have worked.
not sure if the role is necessary anymore / this even needs to be an admin (or `"super_administrator"`).

This looks good.